testing/architecture-test/src/changes/archunit-store/internal-api-nullability.txt

```diff
@@ -2250,6 +2250,29 @@ Class <org.gradle.internal.nativeintegration.jansi.JansiStorageLocator> is not a
+Class <org.gradle.internal.normalization.java.impl.ApiMemberSelector$1> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (ApiMemberSelector.java:0)
+Class <org.gradle.internal.normalization.java.impl.ApiMemberSelector$2> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (ApiMemberSelector.java:0)
+Class <org.gradle.internal.normalization.java.impl.ApiMemberSelector> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (ApiMemberSelector.java:0)
+Class <org.gradle.internal.normalization.java.impl.ApiMemberWriter> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (ApiMemberWriter.java:0)
+Class <org.gradle.internal.normalization.java.impl.ArrayAnnotationValue> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (ArrayAnnotationValue.java:0)
+Class <org.gradle.internal.normalization.java.impl.ClassMember> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (ClassMember.java:0)
+Class <org.gradle.internal.normalization.java.impl.EnumAnnotationValue> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (EnumAnnotationValue.java:0)
+Class <org.gradle.internal.normalization.java.impl.FieldMember> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (FieldMember.java:0)
+Class <org.gradle.internal.normalization.java.impl.InnerClassMember> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (InnerClassMember.java:0)
+Class <org.gradle.internal.normalization.java.impl.Member> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (Member.java:0)
+Class <org.gradle.internal.normalization.java.impl.MethodMember> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (MethodMember.java:0)
+Class <org.gradle.internal.normalization.java.impl.MethodStubbingApiMemberAdapter> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (MethodStubbingApiMemberAdapter.java:0)
+Class <org.gradle.internal.normalization.java.impl.ParameterAnnotationMember> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (ParameterAnnotationMember.java:0)
+Class <org.gradle.internal.normalization.java.impl.SimpleAnnotationValue> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (SimpleAnnotationValue.java:0)
+Class <org.gradle.internal.normalization.java.impl.SortingAnnotationVisitor> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (SortingAnnotationVisitor.java:0)
+Class <org.gradle.internal.normalization.java.impl.TypedMember> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (TypedMember.java:0)
```

Comment:
❗ To fix this, we need to move `:java-language-extensions` to the `:build-logic-commons` level first.